base/src/test/java/io/spine/validate/FieldValidatorShould.java

```diff
@@ -0,0 +1,67 @@
+package io.spine.validate;
+
+import com.google.common.collect.ImmutableList;
+import org.hamcrest.Matchers;
+import org.junit.Test;
+
+import java.util.Collection;
+import java.util.List;
+
+import static com.google.common.collect.ImmutableList.of;
+import static io.spine.test.Verify.assertNotEmpty;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+/**
+ * @author Dmytro Dashenkov
+ */
+public abstract class FieldValidatorShould<V> {
+
+    protected abstract FieldValidator<V>
+    getValidatorForValidatedRequiredField(ImmutableList<V> values);
```

Comment:
Not sure why you need the whole `ImmutableList<V>`, when `Class<V>` seems to be enough.